arch/rv32i/src/pmp.rs

```diff
@@ -99,10 +102,11 @@ impl PMPRegion {
 }
 
 /// Struct storing region configuration for RISCV PMP.
-#[derive(Copy, Clone)]
 pub struct PMPConfig {
     regions: [PMPRegion; 8],
     total_regions: usize,
+    is_dirty: Cell<bool>,
+    last_configured_for: MapCell<AppId>,
```

Comment:
Could be useful to document these fields.

Reply:
Done.